src/libraries/Common/src/Interop/Windows/Advapi32/Interop.TRUSTED_DOMAIN_FULL_INFORMATION.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Advapi32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct TRUSTED_DOMAIN_FULL_INFORMATION
+        {
+            public TRUSTED_DOMAIN_INFORMATION_EX Information;
+            internal TRUSTED_POSIX_OFFSET_INFO PosixOffset;
+            public TRUSTED_DOMAIN_AUTH_INFORMATION AuthInformation;
```

Comment:
```suggestion
            internal TRUSTED_DOMAIN_INFORMATION_EX Information;
            internal TRUSTED_POSIX_OFFSET_INFO PosixOffset;
            internal TRUSTED_DOMAIN_AUTH_INFORMATION AuthInformation;
```

Just to keep them consistent. The type itself is `internal` so this is a safe operation.